app/assets/stylesheets/modules/digital.scss

```diff
@@ -1,42 +1,163 @@
-
 /* Digital pack */
-.digital-cta {
-    padding-top: $gs-baseline*2;
-    padding-bottom: $gs-baseline*3;
+
+.digital-pack {
```

Comment:
Is there a more component driven way of doing this? It's a good idea to avoid scoping styles based on a body class name.